go.mod

```diff
@@ -403,3 +406,5 @@ replace github.com/toolkits/concurrent => github.com/niean/gotools v0.0.0-201512
 
 // this is a fork which addresses a performance issues due to go routines
 replace dubbo.apache.org/dubbo-go/v3 => dubbo.apache.org/dubbo-go/v3 v3.0.3-0.20230118042253-4f159a2b38f3
+
+replace github.com/dapr/dapr => github.com/italypaleale/dapr v1.6.1-0.20230404165709-453bccb6c007
```

Comment:
Why here? I would understand `tests/certification/go.mod` but we never include and pin dapr/dapr here.

Reply:
I just used the command in the Makefile to add the replace. I can remove it (I don't think it's used anyways)

Reply:
Done